Content.Client/Options/UI/Tabs/AudioTab.xaml

```diff
@@ -49,7 +49,20 @@
                     <Control MinSize="4 0"/>
                 </BoxContainer>
                 <BoxContainer Orientation="Horizontal" Margin="5 0 0 0">
-                    <Label Text="{Loc 'ui-options-ambience-max-sounds'}" />
+                    <Label Text="{Loc 'ui-options-lobby-volume'}" HorizontalExpand="True" />
+                    <Control MinSize="8 0" />
+                    <Slider Name="LobbyVolumeSlider"
+                            MinValue="0"
+                            MaxValue="300"
```

Comment:
Do it really need to go all the way up to 300? Currently, master volume goes up to 100%, MIDI (Instrument) volume goes up to 200%, and ambience goes up to 300%.

Reply:
Yeah I just copied the ambience slider. I have no idea why they all have different max values.

Reply:
I suggest setting it to 200 then maybe

Reply:
yeah set to 200 to match midi I guess